### DISCUSSION TITLE
Compilation performance/overhead

### DISCUSSION THREAD

**Discussion by zlepper** [thomhurst/TUnit#513] [General]
TUnit looks good, especially with the benchmarks. 

However, how does the compile time change due to the source generators? I remember trying to hack something like this, and I just ended up with the compile times exploding instead, which completely negated any benefits from actually running faster?

You might be able to use something like hyperfine to test it :)

**Comment by thomhurst** [comment]
Hey @zlepper - I've just added some `dotnet build` benchmarks to the readme for each major operating system.

The test projects are very minimal at the moment, but it should still be useful.

How's that?

**Comment by zlepper** [comment]
That's exactly what I was looking for, thank you very much! :D
